chapter2/tests/t1/chapter2-t1-two-files.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+set -eo pipefail
+# Args check
+if [[ "$#" -ne 2 ]]; then
+    echo "Error: Two arguments are required"
+    exit 1
+fi
+# Function for delimiter
+print_separator() {
+    printf '%*s\n' "${COLUMNS:-$(tput cols)}" '' | tr ' ' -
+}
+# Function to generate a random string and write to a file with permissions
+gen_rand_str() {
+    local file="$1"
+    openssl rand -base64 20 > "$file" && chmod 700 "$file"
+}
+
+if   [[ ! -r `pwd` ]]; then
```

Comment:
```suggestion
if   [[ ! -r "$(pwd)" ]]; then
```
See: https://stackoverflow.com/a/4708569